Makefile

```diff
@@ -14,10 +14,12 @@ tidy:
 updatedeps:
 	./tool/go run github.com/tailscale/depaware --update tailscale.com/cmd/tailscaled
 	./tool/go run github.com/tailscale/depaware --update tailscale.com/cmd/tailscale
+	./tool/go run github.com/tailscale/depaware --update tailscale.com/cmd/derper
 
 depaware:
 	./tool/go run github.com/tailscale/depaware --check tailscale.com/cmd/tailscaled
 	./tool/go run github.com/tailscale/depaware --check tailscale.com/cmd/tailscale
+	./tool/go run github.com/tailscale/depaware --check tailscale.com/cmd/derper
```

Comment:
yeah, multiple args works:

```
bradfitz@tsdev:~/src/tailscale.com$ ./tool/go run github.com/tailscale/depaware --check tailscale.com/cmd/tailscale{,d}
bradfitz@tsdev:~/src/tailscale.com$ ./tool/go run github.com/tailscale/depaware --check tailscale.com/cmd/tailscale{,d,e}
2022/08/05 11:17:07 no .go files found for package tailscale.com/cmd/tailscalee
exit status 1
```

Reply:
is your code so slow? :-P

Reply:
"my" code is mostly the go/* packages loading & resolving stuff.

Reply:
silly go team.

I also note that our CI and makefile use different Go toolchains for this, but I'm not going to change that in this PR because I suspect it will slow down the CI check a lot (because stateless nonsense)

Reply:
feel free to submit this as-is

Reply:
I also can't get my local copy of Go to agree with the 1.19 in our CI. Both with my goroot and ./tool/go I see:

```
--- before
+++ after
@@ -83,7 +83,7 @@
         github.com/klauspost/compress/flate                          from nhooyr.io/websocket
         github.com/klauspost/compress/fse                            from github.com/klauspost/compress/huff0
         github.com/klauspost/compress/huff0                          from github.com/klauspost/compress/zstd
-        github.com/klauspost/compress/internal/cpuinfo               from github.com/klauspost/compress/zstd
+        github.com/klauspost/compress/internal/cpuinfo               from github.com/klauspost/compress/zstd+
         github.com/klauspost/compress/internal/snapref               from github.com/klauspost/compress/zstd
         github.com/klauspost/compress/zstd                           from tailscale.com/smallzstd
         github.com/klauspost/compress/zstd/internal/xxhash           from github.com/klauspost/compress/zstd
```

Not sure what's going on there.

Reply:
Do you have a dirty/extra file/package in your tree?

Reply:
that is, in your working directory that's unknown to git?

Reply:
```
~/repo/tailscale (crawshaw/derperdepaware) $ git status
On branch crawshaw/derperdepaware
nothing to commit, working tree clean
```